static/images/icon-custom-outputs.svg

```diff
@@ -0,0 +1,36 @@
+<svg fill="#777" stroke="#fff" width="100pt" height="100pt" version="1.1" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
```

Comment:
👍 Well I need to find someone who can take all of those icons (except for multilingual, which is not good) and make them look like they all came from the same set.